swap_meet/vendor.py

```diff
@@ -1,2 +1,74 @@
 class Vendor:
-    pass
\ No newline at end of file
+
+    def __init__(self, inventory = None):
+        if inventory == None: 
+            self.inventory = []
+        else:
+            self.inventory = inventory 
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item 
+    
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item 
+
+        return False
+    
+    def get_by_id(self, input_id):
+        for item in self.inventory:
+            if item.id == input_id:
+                return item 
+        
+        return None 
```

Comment:
Nice job with this! Might be fun to try using a list comprehension to accomplish the same task!